zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -18,8 +22,18 @@ use crate::{
     util::Expr,
 };
 use eth_types::{evm_types::GasCost, Field, ToLittleEndian, ToScalar};
-use halo2_proofs::circuit::Value;
+use ethers_core::utils::{get_contract_address, keccak256};
+use gadgets::util::expr_from_bytes;
 use halo2_proofs::plonk::Error;
+use halo2_proofs::{circuit::Value, plonk::Expression};
+use num::Zero;
+
+/// When calculating the contract address we take:
+///
+/// RLP([tx_caller_address, tx_nonce])
+///
+/// Defines the minimum length of the RLP-encoding.
+const MIN_RLP_LEN_CONTRACT_CREATION_INPUT: usize = 23;
```

Comment:
Yes, we could do that, but since this constant is specifically used for the `BeginTx` gadget (and not anywhere else in our codebase) I defined it here. Do you think it still makes sense to move it to params module?

Reply:
Agree on that!  In case we need to add this in some other test files, then let's move it later.